File: 02_loop/loops-python/while_loop.py
#1. Print Numbers from 1 to 10
i = 1
while i <= 10:
    print(i)
    i += 1

#2. Print Numbers from 10 to 1
i = 10 
while i >= 1:
    print(i)
    i -= 1

#3. Print Even Numbers from 2 to 20
i = 2
while i <= 20:
    if i % 2 == 0:
        print(i)
    i += 1

#4. Print Odd Numbers from 1 to 19
i = 1
while i <= 19:
    if i % 2 != 0:
        print(i)
    i += 1

#5. Print the Multiplication Table of 8
i = 1
n = 8
while i <= 10:
    print(f"{n} x {i} = {n * i}")
    i += 1

#6. Find the Sum of Numbers from 1 to 10
sum = 0
i = 1
while i <= 10:
    sum = sum + i
    i += 1
print(sum)

#7. Count from 1 to N
i = 1
num = 10
while i <= num:
    print(i)
    i += 1

#8. Print Squares of Numbers from 1 to 10
i = 1
while i <= 10:
    print(i ** 2)
    i += 1

#9. Print All Numbers Divisible by 4 from 1 to 40
i = 1
while i <= 40:
    if i % 4 == 0:
        print(i)
    i += 1

#10. Count Down from N to 1
n = 10

while n >= 1:
    print(n)
    n -= 1
    

File: 02_loop/loops-js/while_loop.js
//1. Print Numbers from 1 to 10
let a = 1;
while (a <= 10) {
    console.log(a);
    a++;
}

//2. Print Numbers from 10 to 1
let b = 10;
while (b >= 1) {
    console.log(b);
    b--;
}

//3. Print Even Numbers from 1 to 20
let m = 1;
while (m <= 20) {
    if (m % 2 === 0) {
        console.log(m);
    }
    m++;
}

//4. Print Odd Numbers from 1 to 20
let p = 1;
while (p <= 20) {
    if (p % 2 !== 0) {
        console.log(p);
    }
    p++;
}

//5. Print the Multiplication Table of 7
let c = 17;
let d = 1;
while (d <= 10 ) {
    console.log(`${c} x ${d} = ${c * d}`);
    d++;
}

//6. Find the Sum of Numbers from 1 to 10
let e = 1;
let sum = 0;
while (e <= 10) {
    sum = sum + e;
    e++;
}
console.log(sum)

//7. Count from 1 to N
let f = 1;
let n = 8;
while (f <= n) {
    console.log(f);
    f++;
} 

//8. Print Squares of Numbers from 1 to 10
let g = 1;
while (g <= 10) {
    console.log(g ** 2);
    g++;
}

//9. Print All Numbers Divisible by 5 from 1 to 50
let h = 1;
while (h <= 50) {
    if (h % 5 === 0) {
        console.log(h);
    }
    h++;
}

//10. Count Down from N to 1
let j = 5;
while (j >= 1) {
    console.log(j);
    j--;
}